release-notes.md

```diff
@@ -1,3 +1,6 @@
+# 1.5.0 (beta)
+- Add support for contextual data via getContextualData function of the config, meant to be used via async hooks
```

Comment:
I'd maybe just say "can be used with async hooks", I can see this being useful for other contextual data as well that doesn't necessarily need async hooks to make sense.